.env.example

```
@@ -39,10 +46,15 @@ REDIS_HOST=redis
 REDIS_PORT=6379
 
 # Sentry Error Tracking (Optional but recommended for production)
-# Get DSN from: https://sentry.io → Settings → Projects → Client Keys
+# Get DSN from: https://sentry.io -> Settings -> Projects -> Client Keys
 SENTRY_DSN=
+SENTRY_SEND_PII=false
+SENTRY_INCLUDE_LOCAL_VARS=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SENTRY_INCLUDE_LOCAL_VARS key should go before the SENTRY_SEND_PII key

(UnorderedKey)